src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -5,7 +5,7 @@ export const formBenefits = {
   '21P-527EZ': 'Veterans pension benefits',
   '21P-530': 'burial benefits',
   '1010ez': 'health care',
-  '22-0993': 'education benefits',
+  '22-0993': 'opt out',
```

Comment:
It looks like this was switched back by a bad merge, as it was already updated in https://github.com/department-of-veterans-affairs/vets-website/pull/8055/files